src/controllers/common/header/NewsBanner.tsx

```diff
@@ -0,0 +1,73 @@
+import "slick-carousel/slick/slick-theme.css";
+import "slick-carousel/slick/slick.css";
+
+import moment from "moment";
+import React, { useEffect, useState } from "react";
+import Slider from "react-slick";
+import { fetchMediumPosts, MediumPost } from "../../../lib/medium/medium";
+import { classNames } from "../../../lib/react/className";
+import { ExternalLink } from "../../../views/ExternalLink";
+
+export const NewsBanner: React.FC = () => {
+  const freshness = 3; // days
+  const take = 4;
+  const [mediumPosts, setMediumPosts] = useState<MediumPost[]>([]);
+
+  useEffect(() => {
+    fetchMediumPosts("renproject")
+      .then((posts) => {
+        setMediumPosts(posts.slice(0, take));
+      })
+      .catch(console.error);
+  }, []);
+
+  const isNew = (timestamp: number): boolean => {
```

Comment:
Minor improvement - If the function does not use state from the component context, it is better to define it outside of the component to prevent unnecessary assignments / re-renders